packages/paste-core/components/avatar/src/index.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {Text} from '@twilio-paste/text';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {IconSize} from '@twilio-paste/style-props';
+import {getComputedTokenNames} from './utils';
+
+export interface AvatarProps {
+  size: IconSize;
+}
+export interface AvatarImageProps {
+  src: string;
+  alt: string;
+}
+
+const AvatarContext = React.createContext<AvatarProps>({size: 'sizeIcon10'});
+
+export const AvatarImg: React.FC<AvatarImageProps> = ({src, alt, ...props}) => {
+  const {size} = React.useContext(AvatarContext);
+  return <Box {...safelySpreadBoxProps(props)} as="img" alt={alt} maxWidth="100%" src={src} size={size} />;
```

Comment:
I think we should add `objectFit: 'cover'` and tweak `maxWidth and size` to `width: 100%, height: 100%`. This will allow us to get rid of the context provider, as the avatar image would just be sized to the container automatically.

Reply:
It's not supported in IE11 unfortunately

Reply:
No longer needed as we've removed the need for the context